.gitignore

```diff
@@ -21,6 +21,9 @@ bld/
 [Oo]bj/
 msbuild.log
 
+# Visual Studio 2015
+.vs/
+
 # Roslyn stuff
```

Comment:
Good point. Our documentation still suggests you can use VS 2015 Preview. We may be able to bump that requirement, but I'm not sure I would  tie it to this PR. 

How about just changing "Roslyn stuff" to "Visual Studio 2015 Pre-CTP6" so that it's clear when Pre-CTP6 is long gone that this can be deleted?